app/tests/spec/lib/fxa-client.js

```diff
@@ -821,7 +779,7 @@ function (chai, $, sinon, FxaClient, p, testHelpers, FxaClientWrapper, AuthError
         sinon.stub(realClient, 'accountUnlockResendCode', function () {
           return p();
         });
-        var triesLeft = Constants.ACCOUNT_UNLOCK_RESEND_MAX_TRIES;
+        var triesLeft = Constants.EMAIL_RESEND_MAX_TRIES - 1;
 
         // exhaust all tries
         var promises = [];
```

Comment:
The below assertion is passing because `realClient.accountUnlockResendCode.called.count` does not exist and is `undefined`. DOH. That should have been `realClient.accountUnlockResendCode.callCount`.